codespan/src/file.rs

```diff
@@ -330,7 +330,7 @@ where
 }
 
 /// A file that is stored in the database.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Hash, PartialEq, Eq)]
```

Comment:
Does it make sense to compare files like this? `Eq` will be expensive and I'm not sure how useful it will be.

Reply:
Yeah, I'm not sure about having these impls. 🤔